substrate/frame/revive/src/wasm/mod.rs

```diff
@@ -56,7 +57,7 @@ use sp_runtime::DispatchError;
 #[codec(mel_bound())]
 #[scale_info(skip_type_params(T))]
 pub struct WasmBlob<T: Config> {
```

Comment:
Shouldn't the `WasmBlob` be renamed? (: (Not necessarily in this PR of course.)

Reply:
Yes there is a lot of renaming to do. I am pushing this of because this will be a nasty PR.